src/components/pages/Item/ExportAnalyticsPage.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { Container, Stack, Typography } from '@mui/material';
+
+import SectionTitle from '@/components/common/SectionTitle';
+import ExportData from '@/components/space/functionality/ExportData';
+import { useAnalyticsTranslation } from '@/config/i18n';
+
+const ExportAnalyticsPage = (): JSX.Element => {
+  const { t } = useAnalyticsTranslation();
+
+  return (
+    <Container>
+      <Stack paddingY={2} spacing={2} alignItems="center">
+        <SectionTitle title={t('EXPORT_ANALYTICS_TITLE')} />
+        <Typography variant="body1" whiteSpace="pre-line">
```

Comment:
I see you have a Container already wrapping the content. But ideally the text should not be longer than 80 char, between 50 and 80 is nice.